src/ServiceControlInstaller.PowerShell/ServiceControlInstaller.PowerShell.csproj

```diff
@@ -13,7 +13,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Update="ServiceControlInstaller.Powershell.dll-help.xml" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
@mikeminutillo we checked the file and we have never updated the help text with the new cmdlets. So we went ahead and removed it

Reply:
I thought we did update the help files when we added new cmdlets. Maybe this is a different file?

Reply:
I did a full text search in the file and they where not present. They where also not present in the module anymore. In addition to that you need to download an archived code plex project to generate that helpfile which I can't remember we did when doing the split

Reply:
I was wrong, never mind https://github.com/Particular/ServiceControl/commit/fcfd57c802a5a722dafd38e0e981ce07a68351da. So I bring this back then